packages/dds/merge-tree/src/localReference.ts

```diff
@@ -17,6 +17,18 @@ import {
 	refTypeIncludesFlag,
 } from "./referencePositions";
 
+export enum SlidingPreference {
```

Comment:
should have a top-level doc comment. maybe something like

Dictates the preferential direction for a {@link ReferencePosition} to slide in a merge-tree.